### ISSUE TITLE
Users SBAT see transaction feeRecipient field on blockscout

### ISSUE THREAD

**Issue by timmoreton** [celo-org/celo-monorepo#998]

**Comment by jcortejoso** [comment]
Hi! Would this view in the `transaction` page be good?
![image](https://user-images.githubusercontent.com/5635989/65449135-9c1b1200-de3a-11e9-86ac-6f00c6b009cf.png)

**Comment by timmoreton** [comment]
LGTM!  Let's make the label "Fee Recipient"

**Comment by diminator** [comment]
Fee is expressed as `Ether`. Should be `cGLD`, right?

**Comment by diminator** [comment]
Not sure about the `USD` conversion?

**Comment by jcortejoso** [comment]
@diminator the correct unit is addressed by #999. Also I have removed the Dollar conversion for now.
Finally I have added the new fields to the rpc endpoint `/api?module=transaction` of blockscout:
```
╰─ curl -s -X GET "http://localhost:4000/api?module=transaction&action=gettxinfo&txhash=0xddc5f43efa3ab4a211301d9cff293f52f79524326a37308368bf14805bc00fd5" -H "accept: application/json" | jq .


{
  "status": "1",
  "result": {
    "value": "0",
    "to": "0x88f24de331525cf6cfd7455eb96a9e4d49b7f292",
    "timeStamp": "1570612328",
    "success": true,
    "next_page_params": null,
    "logs": [
      {
        "topics": [
          "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
          "0x000000000000000000000000748d5ca49510379399b12b319b27be915bc04a6e",
          "0x000000000000000000000000d690d02e5a70cadb5ba3fa1be7ad2a50159726e0",
          null
        ],
        "index": "0",
        "data": "0x00000000000000000000000000000000000000000000000006ccd46763f10000",
        "address": "0x88f24de331525cf6cfd7455eb96a9e4d49b7f292"
      },
      {
        "topics": [
          "0xe5d4e30fb8364e57bc4d662a07d0cf36f4c34552004c4c3624620a2c1d1c03dc",
          null,
          null,
          null
        ],
        "index": "1",
        "data": "0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000125f5f43454c4f5f494e564954455f54585f5f0000000000000000000000000000",
        "address": "0x88f24de331525cf6cfd7455eb96a9e4d49b7f292"
      }
    ],
    "input": "0xe1d6aceb000000000000000000000000d690d02e5a70cadb5ba3fa1be7ad2a50159726e000000000000000000000000000000000000000000000000006ccd46763f10000000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000125f5f43454c4f5f494e564954455f54585f5f0000000000000000000000000000",
    "hash": "0xddc5f43efa3ab4a211301d9cff293f52f79524326a37308368bf14805bc00fd5",
    "gasUsed": "196918",
    "gasPrice": "200",
    "gasLimit": "256658",
    "gasFeeRecipient": "0xe7c7177b6e5418f27e435f96dbf3f7edae41c133",
    "gasCurrency": "0x88f24de331525cf6cfd7455eb96a9e4d49b7f292",
    "from": "0x748d5ca49510379399b12b319b27be915bc04a6e",
    "confirmations": "1359",
    "blockNumber": "239780"
  },
  "message": "OK"
}
```
